test/testdrive/numeric.td

```diff
@@ -37,7 +37,7 @@ a
 # statement ok
 > CREATE OR REPLACE MATERIALIZED VIEW numeric_cast_ok AS
   SELECT 1
-  WHERE mz_now() < 18446744073709551615::numeric(38,0);
```

Comment:
Why'd this need to change?

Reply:
I don't think it needed to change, but I was just reverting all the test changes from the original commit: https://github.com/MaterializeInc/materialize/commit/a88775504bf263ecd212d4180c4c0bedf8904f54#diff-11409f579db3435619b63ce798090baa8677f43b9398ee936589941551d14ab2

The original reason I gave was (https://github.com/MaterializeInc/materialize/pull/13996/files#r936948659):
> Previously mz_logical_timestamp() would return Timestamp::MAX in a view which made this true. Now it returns the current timestamp of the timeline it's in making it false.

So now I think `mz_now()` is back to returning `Timestamp::MAX` in a view.